airflow/windows_extensions/__init__.py

```diff
@@ -0,0 +1 @@
+from airflow.windows_extensions.Signals import Signals
```

Comment:
```suggestion
from airflow.windows_extensions.signals import Signals
```

Module names should be lower case by convention.

Reply:
Wait  -- we have Signals and signals.

Don't do that. Put them both in signals.

Reply:
Actually it’s `Signals` and `signal`, but that’s still (more?) confusing

Reply:
Uups, mistake